pkg/analysis/conflictingmarkers/testdata/src/b/b.go

```diff
@@ -0,0 +1,30 @@
+package b
+
+type CustomConflictStruct struct {
```

Comment:
Include a case from the other tests to make sure that enabling custom markers doesn't disable the built in markers